addons/website_portal/controllers/main.py

```diff
@@ -9,6 +9,16 @@
 from odoo.fields import Date
 
 
+def get_records_pager(ids, current):
```

Comment:
IDK... it is only used for website_portal_* module, so there are some logic to put it in ? no ?
It is maybe the first, but probably not the last helper for portal ;)